pom.xml

```diff
@@ -43,7 +43,7 @@
 
 
     <properties>
-        <quarkus.version>1.4.2.Final</quarkus.version>
+        <quarkus.version>1.5.0.Final</quarkus.version>
```

Comment:
When do you think we will release 1.0 of quarkus-hazelcast-client?

Reply:
Since the extension is published as "preview" under Quarkus platform, it'd make sense to publish GA once we get a green light from them

Reply:
Do you have a link for this? Where is `preview` mentioned?

Reply:
I will try to find, I think it was mentioned somewhere in the discussion under original PR

Reply:
https://github.com/quarkusio/quarkus/pull/6758#discussion_r372479954

Reply:
👍 You can ask for the procedure to move out of preview mode.

Reply:
btw, this is how it looks on the site:

<img width="353" alt="image" src="https://user-images.githubusercontent.com/2182533/83034019-13431e80-a038-11ea-9d91-f30f6d1b13e7.png">